waspc/e2e-test/Tests/WaspNewTest.hs

```diff
@@ -0,0 +1,9 @@
+module Tests.WaspNewTest (waspNew) where
+
+import GoldenTest (GoldenTest, makeGoldenTest)
+import ShellCommands
+  ( waspCliNew,
+  )
+
+waspNew :: GoldenTest
+waspNew = makeGoldenTest "waspNew" ((: []) <$> waspCliNew)
```

Comment:
Woah woah doing some magic here :D :P.
Btw, just in case you didn't know, you could also have done
```hs
waspNew = makeGoldenTest "waspNew" (singleton <$> waspCliNew)
```
or
```hs
waspNew = makeGoldenTest "waspNew" (pure <$> waspCliNew)
```

Reply:
Arg yeah lol `pure` muuuch easier haha thnx